src/olympia/addons/views.py

```diff
@@ -735,6 +736,39 @@ def get_queryset(self):
         return qs.filter(addon=self.get_addon_object())
 
 
+class AddonReviewNotesViewSet(RetrieveModelMixin, ListModelMixin,
+                              GenericViewSet):
+    class AddonViewSetNoPublic(AddonViewSet):
+        permission_classes = [
+            AnyOf(AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted),
+        ]
+    permission_classes = [
+        AnyOf(AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted),
+    ]
+    serializer_class = ReviewNotesSerializer
+    queryset = Version.unfiltered.all()
+
+    def get_addon_object(self):
```

Comment:
If we could do that refactor and figure out a way to test the mixin separately, we could stop re-doing the same tests over and over just for different views - we just need to have a test to make sure the views are depending on the mixin.